resources/lang/de/Template.properties

```diff
@@ -739,6 +739,9 @@ storeSpecialTop = "Top-Artikel"
 termsAndConditions = "AGB"
 termsAndConditionsMetaDescription = ""
 ; termsAndConditions - AGB
+; warranty - Gewährleistung
+warranty = Gewährleistung
```

Comment:
Du hast hier keinen Prefix. Könnte sein, dass dadurch der Wert nicht in der Mehrsprachigkeit auftaucht. Bitte nochmal prüfen. Vermutlich wäre `orderHistoryWarranty` besser.